src/test/java/HoverElementsTest.java

```diff
@@ -0,0 +1,100 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class HoverElementsTest {
+    WebDriver driver;
+    Utils utils;
+    List<WebElement> hoversElements;
+    ArrayList<String> list;
+    int countCheck = 0;
+    int countProfil = 0;
+
+    @BeforeMethod
+    public void initTest(){
+        utils = new Utils();
+        utils.setPropertyDriver();
+
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        //open site
+        utils.openSite(driver, Constants.Hovers.URL_ELEMENTS);
+    }
+
+    @Test(invocationCount = 3)
+    public void hoversTestProfil(){
+        //init elements
+        initHover();
+        initList();
+        testProfil();
+    }
+
+    @Test(invocationCount = 3)
```

Comment:
Not good approach to make Tests dependent on each other (you've used counter++). You specified invocationCount = 3 - what if developers changed profiles count from 3 to 2? This test will not be stable and we will need to fix it. If you want to check all profiles, you can do smth like that:
- To make 1 test and forEach cycle. In this cycle you'll hover all elements displayed on page and store text from each element into collection. Than just assert text within your collection (also using forEach cycle for example)

It's just advice for your future test cases. You can keep this tests 'as is' now.

Reply:
done